jerry-main/main-unix.c

```diff
@@ -435,6 +436,11 @@ main (int argc,
                         "Ignoring 'show-regexp-opcodes' option because this feature is disabled!\n");
       }
     }
+    else if (!strcmp ("--start-debug-server", argv[i]))
+    {
+      flags |= JERRY_INIT_DEBUGGER;
+      jerry_port_default_set_log_level (JERRY_LOG_LEVEL_DEBUG);
```

Comment:
https://github.com/jerryscript-project/jerryscript/pull/1557/files#r98863430
This is why.